runtime/src/lib.rs

```diff
@@ -1437,6 +1437,25 @@ impl pallet_vesting::Config for Runtime {
     const MAX_VESTING_SCHEDULES: u32 = 28;
 }
 
+parameter_types! {
+    // Deposit for storing one new item with key size = 32 bytes and value size = 56 bytes
+    pub const DepositBase: Balance = 15 * currency::CENTS + 88 * 6 * currency::CENTS;
+    // Deposit for adding 32 bytes to an already stored item
+    pub const DepositFactor: Balance = 32 * 6 * currency::CENTS;
+    // Max number of multisig signatories
+    pub const MaxSignatories: u16 = 100;
+}
+
+impl pallet_multisig::Config for Runtime {
+    type Event = Event;
+    type Call = Call;
+    type Currency = Balances;
+    type DepositBase = DepositBase;
+    type DepositFactor = DepositFactor;
+    type MaxSignatories = MaxSignatories;
+    type WeightInfo = pallet_multisig::weights::SubstrateWeight<Runtime>;
```

Comment:
perhaps use the benchmarks we will compute instead:
```suggestion
    type WeightInfo = weights::pallet_multisig::SubstrateWeight<Runtime>;
```